### PR TITLE
Themetoggler:added daisy UI theme toggler.

### PR DESCRIPTION
**Pull request by AMANVISHWAKARMA27**
Fixes : #19

Added daisy UI theme toggler to toggle theme between dark and light.

image , 
Screenshot from 2024-10-03 12-10-45

<details>
<summary> Dark Theme </summary>

image

</details>

<details>
<summary> Light Theme </summary>

image

</details>

### CODE REVIEW

**Comment by AMANVISHWAKARMA27** [comment]
@shuklamaneesh23 , please review the PR.

File: src/app/page.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
 "use client";
 
 import { useState, useEffect } from "react";
-import { parseStream } from "../utils/streaming";
+import ThemeToggler from "@/components/themetoggler"; // Import the ThemeToggler component
```
Comment (shuklamaneesh23) [inline src/app/page.js:4]: why did you removed line no.4?

**Review by shuklamaneesh23 — COMMENTED** [review]
Great Work @AMANVISHWAKARMA27 ,as I told you earlier in the previous PR that change the text colour of input boxes to black in the first two in light mode.
Everything else looks good to me!

File: src/app/page.js (hunk 1 above)
Comment (AMANVISHWAKARMA27) [reply src/app/page.js]: Fixed!!

**Comment by AMANVISHWAKARMA27** [comment]
@shuklamaneesh23 , the text is already black as i see in the local.

image

**Comment by shuklamaneesh23** [comment]
@AMANVISHWAKARMA27 ,LGTM!

**Pull request merged**